lldb/test/API/functionalities/data-formatter/swift-unsafe/main.swift

```diff
@@ -0,0 +1,37 @@
+// main.swift
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+// -----------------------------------------------------------------------------
+struct HasInt {
+  var value: Int
+  
+  init(_ x: Int) {
+    value = x
+  }
+}
```

Comment:
This struct is unused?

You need to test with more data types. Bigger structs, classes, existentials, enums, ...